### ISSUE TITLE
Version 0.3.24 not on dockerhub

### ISSUE THREAD

**Issue by wielerprono** [xury77/AirCon#9]
Hello,
I'm trying to do the docker-compose install, but it fails because it cannot find the image with version 0.3.24
When I go check on docker hub that seems correct because the latest version there is 0.3.22
Can you update your image please?
Thanks!

**Comment by xury77** [comment]
Yes. Since 0.3.23 images are on github. So temporally you can fix it in your docker compose file at row 13
from: image: xury77/aircon:0.3.24
to: image: ghcr.io/xury77/aircon:0.3.24
